discord_bot/discord_bot.py

```diff
@@ -36,7 +69,8 @@ def load_config():
         with open(CONFIG_FILE_PATH) as f:
             return json.load(f)
 
-    def get_gateway(self, **kwargs):
+    @property
+    def get_gateway(self):
```

Comment:
We don't need to use getters/setters, that's only if you need some advanced behavior every time you retrieve/store a value. This is a pretty simple thing that we only want to occur once, so a function is better suited for it.